MacOSExample/src/basic/longPress/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { StyleSheet, View } from 'react-native';
+import { Gesture, GestureDetector } from 'react-native-gesture-handler';
+
+export default function LongPressExample() {
+  const g = Gesture.LongPress()
+    .onBegin(() => console.log('onBegin'))
+    .onStart(() => console.log('onStart'))
```

Comment:
Why no `onEnd`?

Reply:
I've thought that `onFinalize` with `success` will be enough. Anyways, I thought of rewriting this example a bit so that it provides more visual experience than looking at console logs. I'll get back here when I finish 😅

Reply:
I've changed this example, let me know what do you think! ([9cfd459](https://github.com/software-mansion/react-native-gesture-handler/pull/3014/commits/9cfd4598c8ca150d649087bc9c8a558e4d98ebbe))